File: 1. Iniciante/URI1010.py
cod,qtd,valor = input().split(" ")

cod1 = int(cod)
qtd1 = int(qtd)
valor1 = float(valor)

cod,qtd,valor = input().split(" ")

cod2 = int(cod)
qtd2 = int(qtd)
valor2 = float(valor)

x = (qtd2*valor2) + (qtd1*valor1)

print("VALOR A PAGAR: R$ %.2f" %x)

File: 1. Iniciante/URI1010.cpp
#include <iostream>
#include <cstdio>


using namespace std;

int main() {
    int cod1, num1, cod2, num2;
    double valor1, valor2 , TOTAL;

    cin >> cod1 , cin >> num1, cin >> valor1;
    cin >> cod2 , cin >> num2, cin >> valor2;


    TOTAL = (num1 * valor1) + (num2 * valor2);

    printf("VALOR A PAGAR: R$ %.2lf\n", TOTAL);


    return 0;
}
